ex2.sh

```diff
@@ -0,0 +1,29 @@
+# q1:
+ls -S
+
+# a : cat /etc/services
+
+# b:ls -S /etc/ | head -1
+    cat /etc/services
+
+# q2:
+ls -S -r
```

Comment:
The command is correct, but you can combine the flags -S and -r to simplify the command